entrypoint_test.go

```diff
@@ -16,51 +16,45 @@ example val
 another green	
 `
 	replacements := []Replacement{
-		Replacement{
+		{
 			Key:   "TEST",
 			Value: "val",
 		},
-		Replacement{
+		{
 			Key:   "FIELD",
 			Value: "green",
 		},
 	}
 	results := UpdateConfigContent([]byte(original), replacements)
 
 	if string(results) != expected {
-		t.Error("Results to not match expected. Results:", results)
-		t.FailNow()
+		t.Fatal("Results to not match expected. Results:", results)
 	}
 }
 
 func TestBuildReplacementsFromEnv(t *testing.T) {
 	// Test failure for required env var
 	_, err := BuildReplacementsFromEnv()
 	if err == nil {
-		t.Error("BuildReplacementsFromEnv should have failed because no env vars have been set")
-		t.FailNow()
+		t.Fatal("BuildReplacementsFromEnv should have failed because no env vars have been set")
 	}
 
 	setRequiredEnvVars()
 
 	replacements, err := BuildReplacementsFromEnv()
 	if err != nil {
-		t.Error(err)
-		t.FailNow()
+		t.Fatal(err)
 	}
 
-	replacementsCount := len(replacements)
-	if replacementsCount != 6 {
-		t.Error("Replacements did not have enough entries, only found", replacementsCount, "but expected 6")
-		t.FailNow()
+	if want, got := 7, len(replacements); want != got {
```

Comment:
Why did it change from 6 to 7?

Reply:
There was a bug in setting the default, where it wasn't actually setting it.

Reply:
So `DNS_PROVIDER` is now in the array when it wasn't previously.